### DISCUSSION TITLE
Particle (water droplet) age

### DISCUSSION THREAD

**Discussion by MarioCruzM** [firemodels/fds#13941] [Q&A]
I've been reading the FDS guide on topics related to the "lifetime/age" of particles. In the specific case of water droplets, so far I understand that, as long as I don't have suppression fire on and my fire is not located on the floor, the age of the water droplets -specifically those that reach and can accumulate on the floor—doesn't really interfere with the maximum HRR. Is my understanding correct, or are there any circumstances in which this actually has an impact?

**Comment by rmcdermo** [comment]
If `SUPPRESSION=F` and you have a well-ventilated fire, then you should expect to achieve your prescribed HRR.

**Comment by drjfloyd** [comment]
If you are specifying your fire with HRRPUA or MLRPUA and do not give an E_COEFFICEINT, then water on the floor is not going to impact the mass flux of fuel at the fire.
